### DISCUSSION TITLE
Changing tag styling, not quite working

### DISCUSSION THREAD

**Discussion by carljoseph** [ebullient/obsidian-theme-ebullientworks#20] [Q&A]
Trying to make tags into a "pill style". I've got most of it working, except it looks different in editor/read mode.

All looking good in preview/read mode. Except in editor mode the # and _text_ portion of the tag are separated. Any ideas?

**Here's how it currently looks**
<img width="490" alt="Screen Shot 2022-12-03 at 5 51 54 pm" src="https://user-images.githubusercontent.com/4588120/205428928-76d4febe-2160-4b9e-b2e4-b13bd0cf6783.png">

**Here's my CSS snippet**
```
a.tag,
.editor .cl-hashtag,
.markdown-source-view.mod-cm6 .cm-hashtag,
.cm-s-obsidian .cm-formatting.cm-hashtag,
.cm-s-obsidian .cm-quote.cm-hashtag,
.cm-s-obsidian .cm-hmd-codeblock.cm-hashtag {
  background: #e3e3e3;
  padding: 2px 8px;
  margin: 0 0;
  border-radius: 14px;
  text-align: center;
  color: var(--red-5);
  border: var(--red-5) 0.5px solid;
}

a.tag:hover {
  background: #e3e3e3;
  color: var(--blue-5);
}
```

**Comment by ebullient** [comment]
I'll tuck my tag styling under style settings so you can turn it off.. which may make it easier for you to customize using core styles.

**Reply by ebullient** [reply]
I've released that change in 0.6.1.
Now that the css for tags has been regrouped (and can be turned off), it may give you some ideas for how to adjust what you've tried.. 

https://github.com/ebullient/obsidian-theme-ebullientworks/blob/938d0b005687bafa2c0931047d1ce57cb7d9eb7e/theme.css#L912
